### PR TITLE
[#87323012][1pts] Search box

### PR DESCRIPTION
**Pull request by zalapeach**
* Adds styles for not found posts
* Adds not found text

Search box [Finishes #87323012]

### CODE REVIEW

File: app/assets/stylesheets/modules/posts.scss [hunk 1]
```diff
@@ -4,10 +4,23 @@
   text-align: left;
   width: 70%;
 
-  .posts-search-title {
-    margin: 25px 50px 0;
+  .posts-search {
+    border-top: 1px solid $gray-medium-light;
+    margin-top: 35px;
+    padding: 50px;
```
Comment (alexesba) [inline app/assets/stylesheets/modules/posts.scss]: Could you Please, Sort the css properties.

**Comment by alexesba** [comment]
Beside my comment LGTM.

**Comment by zalapeach** [comment]
Done @alexesba thanks for your time

File: app/assets/stylesheets/modules/posts.scss [hunk 2]
```diff
@@ -4,10 +4,23 @@
...
-    margin: 25px 50px 0;
+  .posts-search {
+    border-top: 1px solid $gray-medium-light;
+    color: $gray-medium-light;
+    margin-top: 35px;
+    padding: 50px;
+    text-align: center;
 
-    @include mq-mobile() { margin: 25px 20px 0; }
+    a { color: $gray-medium-light; }
```
Comment (iszandro) [inline app/assets/stylesheets/modules/posts.scss]: I would change this selector for a class selector maybe `.posts-search-link` or something like that

**Comment by zalapeach** [comment]
Done @iszandro thanks!

**Comment by heridev** [comment]
LGTM

**Pull request merged**